### PR TITLE
br: update some faq from oncall 

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update some br docs and add some faq.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-use-cases.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ BR 可以直接将命令下发到 TiKV 集群来执行备份和恢复，不依
 
 ### 备份前的准备工作
 
-如果你使用的是 TiDB v4.0.8 及以上版本，相应版本的 BR 工具已支持自适应 GC。你只需要将 `backupTS` 注册到 PD 的 `safePoint`，保证 `safePoint` 在备份期间不会向前移动，即可避免手动设置 GC。
+如果你使用的是 TiDB v4.0.8 及以上版本，相应版本的 BR 工具已支持自适应 GC。会自动将 `backupTS`(默认是最新 PD timestamp) 注册到 PD 的 `safePoint`，保证 `safePoint` 在备份期间不会向前移动，即可避免手动设置 GC。
```
Comment (overvenus) [inline br/backup-and-restore-use-cases.md:81]:
```suggestion
如果你使用的是 TiDB v4.0.8 及以上版本，相应版本的 BR 工具已支持自适应 GC。会自动将 `backupTS`(默认是最新 PD timestamp) 注册到 PD 的 `safePoint`，保证 TiDB 的 GC Safe Point 在备份期间不会向前移动，即可避免手动设置 GC。
```

**Review by overvenus — COMMENTED** [review]
Rest LGTM

**Comment by TomShawn** [comment]
@3pointer Please update this PR and resolve the conflicts, thanks~

**Review by amyangfei — APPROVED** [review]

File: br/backup-and-restore-use-cases.md [hunk 2]
```diff
@@ -78,6 +78,8 @@ BR 可以直接将命令下发到 TiKV 集群来执行备份和恢复，不依
 
 ### 备份前的准备工作
 
+如果你使用的是 TiDB v4.0.8 及以上版本，相应版本的 BR 工具已支持自适应 GC。会自动将 `backupTS`(默认是最新 PD timestamp) 注册到 PD 的 `safePoint`，保证 TiDB 的 GC Safe Point 在备份期间不会向前移动，即可避免手动设置 GC。
```
Comment (TomShawn) [inline br/backup-and-restore-use-cases.md:81]: We can add this back in release-4.0.
```suggestion
BR 工具已支持自适应 GC，会自动将 `backupTS`（默认是最新的 PD timestamp）注册到 PD 的 `safePoint`，保证 TiDB 的 GC Safe Point 在备份期间不会向前移动，即可避免手动设置 GC。
```

File: br/backup-and-restore-use-cases.md [hunk 3]
```diff
@@ -203,7 +205,7 @@ BR 会在备份结束时输出备份总结到控制台。
 
 #### 性能调优
 
-如果 TiKV 的资源使用没有出现明显的瓶颈（例如[备份过程中的运行指标](#备份过程中的运行指标)中的 **Backup CPU Utilization** 最高为 `1500%` 左右，**IO Utilization** 普遍低于 `30%`），可以尝试调大 `--concurrency` 参数以进行性能调优。该方法不适用于存在许多小表的场景。
+如果 TiKV 的资源使用没有出现明显的瓶颈（例如[备份过程中的运行指标](#备份过程中的运行指标)中的 **Backup CPU Utilization** 最高为 `1500%` 左右，**IO Utilization** 普遍低于 `30%`），可以尝试调大 `--concurrency`(默认是 4) 参数以进行性能调优。该方法不适用于存在许多小表的场景。
```
Comment (TomShawn) [inline br/backup-and-restore-use-cases.md:208]:
```suggestion
如果 TiKV 的资源使用没有出现明显的瓶颈（例如[备份过程中的运行指标](#备份过程中的运行指标)中的 **Backup CPU Utilization** 最高为 `1500%` 左右，**IO Utilization** 普遍低于 `30%`），可以尝试调大 `--concurrency`（默认是 4）参数以进行性能调优。该方法不适用于存在许多小表的场景。
```

File: br/backup-and-restore-faq.md [hunk 4]
```diff
@@ -90,6 +90,10 @@ TiCDC 可以通过配置项中的 [`filter.rules`](https://github.com/pingcap/ti
 
 可以使用 kubectl 执行 `kubectl -n ${namespace} get bk ${name}` 以获得上次 BR 备份 `commitTs` 字段，该字段的内容可作为 `--lastbackupts` 使用。
 
+## BR backupTS 如何转化成 Unix 时间?
+
+BR backupTS 默认情况下是在备份开始前，从 PD 获取到的最新 timestamp。可以采用 pd-ctl tso `timestamp` 来解析，获得精确值，也可以通过 `backupTS >> 18` 来快速获取估计值。
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:95]:
```suggestion
BR `backupTS` 默认是在备份开始前，从 PD 获取到的最新时间戳。可以使用 `pd-ctl tso timestamp` 来解析该时间戳，以获得精确值，也可以通过 `backupTS >> 18` 来快速获取估计值。
```

**Pull request merged**